src/yang/ietf-bgp.yang

```diff
@@ -344,9 +342,7 @@ module ietf-bgp {
           }
 
           leaf remote-port {
-            type inet:port-number {
-              range "0..65535";
-            }
+            type inet:port-number
```

Comment:
Missing a semicolon.

Reply:
Fixed and will update pull request with it.